DatabaseWorkshop.md

```diff
@@ -8,6 +27,11 @@ Now we will show you how to use our ORM (Object Relational Mapping) library, cal
 
 This is a follow on tutorial to our [ToDoBackend tutorial](https://github.com/IBM/ToDoBackend). Please complete that before proceeding with this tutorial.
 
+**Homebrew** is also needed, to install homebrew paste this into your terminal and hit Enter:
```

Comment:
Homebrew should be capitalized. Maybe just say "run the following Terminal command"?